test/Microsoft.Azure.Relay.UnitTests/WebSocketTests.cs

```diff
@@ -0,0 +1,232 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Relay.UnitTests
+{
+    using System;
+    using System.Net;
+    using System.Net.WebSockets;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Xunit;
+
+    public class WebSocketTests : HybridConnectionTestBase
+    {
+        const string TestAcceptHandlerResultHeader = "X-TestAcceptHandlerResult";
+        const string TestAcceptHandlerStatusCodeHeader = "X-TestAcceptHandlerStatusCode";
+        const string TestAcceptHandlerStatusDescriptionHeader = "X-TestAcceptHandlerStatusDescription";
+        const string TestAcceptHandlerSetResponseHeader = "X-TestAcceptHandlerSetResponseHeader";
+        const string TestAcceptHandlerDelayHeader = "X-TestAcceptHandlerDelay";
+
+        public  WebSocketTests() : base()
```

Comment:
> : base() [](start = 32, length = 9)

Nit: Again " : base()" is optional.  (My general guideline is "If optional text doesn't change anything from the language default then don't add that text".  Or maybe "if the optional text doesn't change anything from a *well-known* language default...".  In general for me, the fewer characters my eyes and brain have to see, the better :)